src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -23,7 +23,7 @@ import { AnalysisError } from "./AnalysisError";
 import { getI18N } from "../../../utilities/i18n-utilties";
 import { SPACE_MD } from "../../../styles/spacing";
 import styled from "styled-components";
-
+import { navigate } from "@reach/router";
 import {
   AnalysisContext,
   AnalysisProvider
```

Comment:
`AnalysisProvider` is never used

Reply:
Fixed in dc1a014b789d7270dd33c9e420a21a122a5e796a